src/css/structure/jquery-ui.css

```diff
@@ -1126,7 +1126,8 @@ a.ui-button:focus {
     height : 16px;
 }
 
-.ui-icon,
+
+/*.ui-icon,
```

Comment:
The issue comes from the existence of this file and the fact that it's being combined with the rest of the structure files. I believe it should be removed entirely.